packages/components/nodes/agents/OpenAIAssistant/OpenAIAssistant.ts

```diff
@@ -158,7 +160,7 @@ class OpenAIAssistant_Agents implements INode {
         }
 
         const credentialData = await getCredentialData(assistant.credential ?? '', options)
-        const openAIApiKey = getCredentialParam('openAIApiKey', credentialData, nodeData)
+        const openAIApiKey = getCredentialParam('openAIApiKey', credentialData, nodeData) || process.env.AAI_DEFAULT_OPENAI_API_KEY
```

Comment:
We need to check if this imports ALL of our assistants and allows people to edit assistants on our account. Please create an assistant in one account, and see if you can see it in another different account or if its protected